code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Food-Drinks.dm

```diff
@@ -964,6 +964,19 @@
 	glass_name = "glass of cream"
 	glass_desc = "Ewwww..."
 
+/datum/reagent/drink/milk/pigmilk
+	name = "Strange Milk"
+	id = "pigmilk"
+	description = "An opaque white liquid produced by the mammary glands of mammals...the scent is slightly different however"
```

Comment:
A quick google makes me think that this would be impossible to tell apart from normal milk

Reply:
![opera_psuadss7st](https://user-images.githubusercontent.com/43487478/50622463-19c07b80-0ec2-11e9-851f-eac3417ec934.png)